src/components/Content/index.tsx

```diff
@@ -65,38 +61,57 @@ export const Content: React.VFC = () => {
   return (
     <PageGrid>
       <Header />
-      <Switch>
-        <Route path={routes.task} component={Task} />
-        <Route path={routes.configurePatch} component={ConfigurePatch} />
-        <Route exact path={routes.patch} component={VersionPage} />
-        <Route path={routes.version} component={VersionPage} />
-        <Route path={routes.jobLogs} component={JobLogs} />
-        <Route path={routes.hosts} component={Hosts} />
-        <Route path={routes.host} component={Host} />
-        <Route path={routes.myPatches} component={MyPatches} />
+      <Routes>
+        <Route path={routes.task} element={<Task />}>
```

Comment:
Could we put these routes in alphabetical order while we are changing things here?